app/src/main/res/layout/item_exposure.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <data>
+
+        <import type="java.util.Date" />
+
+        <import type="android.view.View" />
+
+        <variable
+            name="viewModel"
+            type="org.covidwatch.android.ui.exposures.ExposuresViewModel" />
+
+        <variable
+            name="item"
+            type="org.covidwatch.android.data.CovidExposureInformation" />
+    </data>
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:gravity="center_vertical"
+        android:onClick="@{()->viewModel.showExposureDetails(item)}"
```

Comment:
Why do you need to prefix with `()->` instead of just the function itself? Understand q more than a functional q

Reply:
This is  data binding syntax that means we insert a lambda function that has no parameters and return `Void`